000 - glossary.md

```diff
@@ -1,39 +1,56 @@
 # Glossary of Terms
 
 This page is intended to be a quick reference to terms that we use throughout
-this guide. These terms are defined within the context of cloud-native software
-systems – more general definitions also exist for more general contexts but are
-not provided here.
+this project.
 
 ---
 
 _Distributed System_: A system that's characterized by its components
 being split up across multiple distinct computers on a network.
 
+_Distributed Tracing_: A method of profiling and monitoring distributed system
```

Comment:
When I define DT I am pretty careful to describe how this form of tracing follows transactions, and particularly follows them across process and machine/infra/network boundaries. My quick take:

```
_Distributed Tracing_: The technique of following – or "tracing" – requests as they traverse distributed systems, representing them as telemetry, and reassembling and analyzing models of the distributed requests.
```